graphql_compiler/schema_generation/sqlalchemy/schema_graph_builder.py

```diff
@@ -126,10 +126,21 @@ def _get_edge_types(tables):
     edge_types = []
     for table in six.itervalues(tables):
         for column in table.get_children():
-            for foreign_key in column.foreign_keys:
+            if len(column.foreign_keys) > 1:
+                raise IllegalSchemaInputError(u'Multiple foreign keys are defined on column {} of '
+                                              u'table {}.'.format(column.key, table.name))
+            if len(column.foreign_keys) > 0:
```

Comment:
Nit: check len(column.foreign_keys) == 1, not inequality. This `if` basically has a dependency on the above condition.